bravado/mapping/operation.py

```diff
@@ -154,8 +156,9 @@ def handle_response(response, op):
     """
     if isinstance(response, requests.models.Response):
         wrapped_response = RequestsLibResponseAdapter(response)
+    elif isinstance(response, fido.fido.Response):
+        wrapped_response = FidoLibResponseAdapter(response)
```

Comment:
With my proposed interfaces we wouldn't need this branching.  The client would return the correct adaptor directly.

Branching is unnecessary, and type checks are not very pythonic.  We can avoid path.